docs/migration/4_0.md

```diff
@@ -719,6 +719,64 @@ The following properties where removed in 4.0:
 - Change detection strategy for this component was set to `OnPush`.
 - Spinner component was added to the template which relys on `isUpdating$` property.
 
+### LogoutGuard
+
+`AuthRedirectService` is a new, required constructor dependency.
```

Comment:
```suggestion
### LogoutGuard
`AuthRedirectService` is a new, required constructor dependency.
```

Reply:
btw this is guard changes, it shouldn't be here. Please move it to other section

Reply:
yup, it looks like mistake -> https://github.com/SAP/spartacus/pull/12737/files#r651275286